fluxcd/bootstrap.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+set -x
+# Override with the `env` key if necessary
+export KUBECONFIG=/etc/rancher/k3s/k3s.yaml
```

Comment:
This makes will only work on k3s. There may be kairos' out there that run different providers with different k8s distributions. On the other hand, I'm not sure how it could be set outside the bundle. In any case, maybe it's better to set it to `k3s.yaml` if not already set.

Reply:
The environment variables for Flux can be defined in the cloud-config, that line just sets a default. Here's how:

```yaml
flux:
  env:
    KUBECONFIG: /home/user/.kube/config
```

This part reads the keys under `env` and sets all variables and overrides the default if a `KUBECONFIG` is set. https://github.com/tyzbit/community-bundles/blob/f66dc04ce10a9826c7e970136f4b8271f8ce5565/fluxcd/bootstrap.sh#L40-L50

I've added an example how to override the KUBECONFIG for non-k3s distros.